app/controllers/api/v1/timesheet_entry_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class Api::V1::TimesheetEntryController < Api::V1::BaseController
+  include Timesheet
+
+  def create
+    return render json: { status: 403, notice: "User is not a project member." }, status: 403 if project_member.blank?
```

Comment:
```suggestion
    return render json: { notice: "User is not a project member." }, status: :forbidden if project_member.blank?
```

This is more readable